packages/synthchron-postprocessor/src/postprocess.ts

```diff
@@ -1,10 +1,22 @@
 import seedrandom from 'seedrandom'
 
-import { Trace } from '@synthchron/simulator'
+import { SimulationLog, Trace } from '@synthchron/simulator'
 import { Configuration, PostprocessingStepType } from '@synthchron/types'
 
 import { insertDuplicate } from './insert'
 
+export const constPostprocessSimulation = (
```

Comment:
I have made the postprocessor work with single traces so this will have to change I guess. Also you have const in the name of the function